spec/support/spec_macros.rb

```diff
@@ -0,0 +1,26 @@
+module Dradis::Plugins::SpecMacros
```

Comment:
namespaced under `Dradis::Plugins` so that there's no chance of naming conflicts with integrations